test/mock_firestore_db.dart

```diff
@@ -276,4 +276,20 @@ class MockFirestoreDb implements FirestoreDb {
   Future<void> guessBrenda(String id, String playerId) {
     return upsertRoom(room.copyWith(brendaGuess: playerId));
   }
+
+  @override
+  Future<void> updateRole(String id, String roleId, bool selected) {
+    Set<String> roles = room.roles;
```

Comment:
Might be safer to clone the set: `Set<String> roles = Set.of(room.roles);`

Reply:
Done